OrbitGl/App.cpp

```diff
@@ -682,6 +688,7 @@ bool OrbitApp::OnLoadSession(const std::string& file_name) {
 
   std::ifstream file(file_path);
   if (file.fail()) {
+    ERROR("Loading session from \"%s\"", file_path);
```

Comment:
maybe propagate this message via result<void,std::string> - in case of failure there is a error message displayed.

Reply:
This returns a bool and the caller (`on_actionOpen_Session_triggered`) does display an error message. Of course they are not the same message, but this is intended as they have different purposes, one as a log (with the description of the exception) and the other as an interactive message to the user.

Reply:
maybe user would benefit from more detailed information, file does not exists/error while reading data from file.

Reply:
So the idea here would be to disambiguate between the `file.fail()` case and the `std::exception` case? 
If yes, I guess that in the `std::exception` case you would just mention that the file is malformed (`e.what()` often makes little sense). But what about the `file.fail()` case? There could be multiple reasons and we would have to actually find out which one it is. Or would we go for a generic "the file cannot be read/written"?

Reply:
And also what message would you use when an exception is caught on write?

Reply:
I would go with generic message "was not able to write into %s" on write unless e.what() makes sense to user.

Reply:
Done. I've kept the error messages very generic, basically distinguishing between "input/output error" and "parsing/serialization error".